conan/cli/commands/remote.py

```diff
@@ -72,9 +72,11 @@ def remote_add(conan_api, parser, subparser, *args):
                            help="Insert the remote at a specific position in the remote list")
     subparser.add_argument("-f", "--force", action='store_true',
                            help="Force the definition of the remote even if duplicated")
+    subparser.add_argument("-t", "--type", choices=["local"],
+                           help="Define the remote type")
```

Comment:
My only gripe with this is that only having "local" as a choice makes it seem in the help output that only "local" is supported, not that it's an opt-in for local remotes. Do you think we could also add a "remote" (or something) default  value?

Reply:
Agree, I haven't thought the UX or the naming at all. There are other cases we might want to consider too.